weaviate/graphql/multitargetbuilder.go

```diff
@@ -174,3 +173,25 @@ func (m *MultiTargetArgumentBuilder) build() string {
 	}
 	return strings.Join(clause, " ")
 }
+
+func (m *MultiTargetArgumentBuilder) togrpc() *pb.Targets {
+	if len(m.targetVectors) > 0 {
+		var weightsForTargets []*pb.WeightsForTarget
+		for i, weights := range m.weights {
+			for _, w := range weights {
+				weightsForTargets = append(weightsForTargets, &pb.WeightsForTarget{
+					Target: m.targetVectors[i],
+					Weight: w,
+				})
+			}
+		}
+		_, combination := m.getCombinationMethod()
```

Comment:
question: is the first return value (`string`) used anywhere?

Reply:
it's used in line 162

Reply:
Right, for building the GraphQL equivalent of the request, see it now